ledger/src/blockstore.rs

```diff
@@ -2731,6 +2732,27 @@ impl Blockstore {
         Ok((entries, num_shreds, slot_meta.is_full()))
     }
 
+    pub fn get_accounts_used_in_range(
+        &self,
+        starting_slot: Slot,
+        ending_slot: Slot,
+    ) -> Result<DashSet<Pubkey>> {
```

Comment:
no reason to return Result here. We are always returning DashSet

Reply:
ah, we can ? return. Nevermind.

Reply:
well, but if we return an error, it seems we probably want to keep going to get all slots. If we have skipped slots do we return Ok<empty> for those slots or do we return an error?

Reply:
how long does this fn take? could also par iter on starting_slot..=ending_slot

Reply:
That's a good point, I hadn't run into a case with missing slots in my range of interest in my testing. We should process the slots we have entries for, and not error out on missing.

Reply:
Time is of course going to depend on the number of slots, as well as how many transactions are in each slot. My range of interest was 600 slots on devnet `[136943500, 136944100]`, which takes about 1.8s or 3ms/slot on average.

I'll test out making the slot loop parallelized as well.

Reply:
Parallelizing the slot loop brings this down (same range) to 214ms.